tmk_core/protocol/ps2_mouse.h

```diff
@@ -75,14 +75,24 @@ __attribute__((unused)) static enum ps2_mouse_mode_e {
  *    1|[                    X movement(0-255)                         ]
  *    2|[                    Y movement(0-255)                         ]
  */
-#define PS2_MOUSE_BTN_MASK 0x07
-#define PS2_MOUSE_BTN_LEFT 0
-#define PS2_MOUSE_BTN_RIGHT 1
-#define PS2_MOUSE_BTN_MIDDLE 2
-#define PS2_MOUSE_X_SIGN 4
-#define PS2_MOUSE_Y_SIGN 5
-#define PS2_MOUSE_X_OVFLW 6
-#define PS2_MOUSE_Y_OVFLW 7
+#define PS2_MOUSE_BTN_MASK 0x07 // 0b0000111
+#define PS2_MOUSE_BTN_LEFT 0    // 0b0000001
+#define PS2_MOUSE_BTN_RIGHT 1   // 0b0000010
+#define PS2_MOUSE_BTN_MIDDLE 2  // 0b0000100
+#define PS2_MOUSE_X_SIGN 4      // 0b0001000
+#define PS2_MOUSE_Y_SIGN 5      // 0b0010000
+#define PS2_MOUSE_X_OVFLW 6     // 0b0100000
+#define PS2_MOUSE_Y_OVFLW 7     // 0b1000000
+
+#ifdef PS2_MOUSE_INVERT_BUTTONS
+/*
+ * LUT for change mouse buttons
+ *
+ * table index is the mouse_report->buttons state from PS2 (after &= PS2_MOUSE_BTN_MASK)
+ * the value replace the mouse_report->buttons before to send to the USB interface
+ */
+static const __attribute__((unused)) uint8_t btns_lut[8] = {0b000, 0b010, 0b001, 0b011, 0b100, 0b101, 0b110, 0b111};
+#endif
```

Comment:
This should be moved to `tmk_core/protocol/ps2_mouse.c`. Also `__attribute__((unused))` shouldnt be required given its always got a call site when PS2_MOUSE_INVERT_BUTTONS is defined.

Reply:
The same that previous one. I put it there to not forget it and the __attribute__((unused)) is because on the testing I coment the line on the .c

@zvecr Tell me wht you prefer. The LUT or the one in https://github.com/qmk/qmk_firmware/pull/9205/commits/7811e9f2bedf281266c489da688fca773eb4e1cd